ml-agents-envs/mlagents/envs/tests/test_brain.py

```diff
@@ -0,0 +1,62 @@
+import logging
```

Comment:
Confirmed that this test passed on develop, aside from checking that nan_to_num wasn't called (because it always is currently).